sdk/core/perf/inc/azure/perf/dynamic_test_options.hpp

```diff
@@ -22,14 +22,18 @@ namespace Azure { namespace Perf {
   class TestOptions {
   private:
     argagg::parser_results m_results;
+    bool m_skipInitial = true;
 
   public:
     /**
      * @brief Create the test options component from the command line parsed results.
      *
      * @param results The command line parsed results.
      */
-    explicit TestOptions(argagg::parser_results results) : m_results(results) {}
+    explicit TestOptions(argagg::parser_results results, bool skipInitial = true)
```

Comment:
Add a comment, what `skipInitial` is doing and what is it for. I am reading your PR, and I still have no idea.
There is a doxygen section above, plus there is a new method that you are adding (`ShouldSkipInitial()`).